### PR TITLE
DOCS-10881: Added partial with note

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/_partials/mq-big-payload.adoc [hunk 1]
```diff
@@ -5,3 +5,9 @@ You can retrieve them with the Anypoint Broker API.
 See 
 xref:mq-apis.adoc#example-retrieve-a-message[Example: Retrieve a Message].
 // end::mqBigPayload[]
+
+// MQ XML Payload
+// tag::mqXMLPayload[]
+If the payload contains XML, Anypoint MQ converts it to a string before sending, which increases the payload size.
```
Comment (prafulrana) [inline modules/ROOT/pages/_partials/mq-big-payload.adoc]: should we provide an example @hannanelson koustub?

Comment (hannanelson) [reply modules/ROOT/pages/_partials/mq-big-payload.adoc]: @kchoudhari-mule ?

Comment (schinthakindims) [reply modules/ROOT/pages/_partials/mq-big-payload.adoc]: Shouldn't this apply for all Media types and not just XML

https://www.mulesoft.org/docs/site/4.1.1/apidocs/org/mule/runtime/extension/api/annotation/param/MediaType.html

ie., XML, CSV, HTML, JSON. etc.,

Comment (hannanelson) [reply modules/ROOT/pages/_partials/mq-big-payload.adoc]: Oh! I didn't get that from the ticket. So all formats are converted to string and increase?

Comment (kchoudhari-mule) [reply modules/ROOT/pages/_partials/mq-big-payload.adoc]: I don't think an example is needed. It is pretty standard XML to String conversion using Jackson.
Yes, it should apply to all media types.

**Review by kchoudhari-mule — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/mq-queues.adoc [hunk 2]
```diff
@@ -93,7 +93,11 @@ From this page, you can:
 
 == Send a Message to a Queue
 
-Message content can be text, CSV (comma-separated values), or JSON. The maximum size of a message is 10 MB. 
+Message content can be text, CSV (comma-separated values), or JSON.
+The maximum size of a message is 10 MB.
+
+//XML Payload
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/mq-queues.adoc]: there's a space between // and XML above

File: modules/ROOT/pages/_partials/mq-big-payload.adoc [hunk 3]
```diff
@@ -5,3 +5,9 @@ You can retrieve them with the Anypoint Broker API.
 See 
 xref:mq-apis.adoc#example-retrieve-a-message[Example: Retrieve a Message].
 // end::mqBigPayload[]
+
+// MQ XML Payload
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/_partials/mq-big-payload.adoc]: Is the MQ necessary in the sections below?

**Review by IsaacEldridge — DISMISSED** [review]

File: modules/ROOT/pages/_partials/mq-big-payload.adoc (hunk 3 above)
Comment (hannanelson) [reply modules/ROOT/pages/_partials/mq-big-payload.adoc]: Those are comments, so spaces don't really matter. But I do need to change that comment.

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**